arrow/internal/flight_integration/scenario.go

```diff
@@ -2228,6 +2228,7 @@ func getQuerySchema() *arrow.Schema {
 				IsSearchable(true).
 				CatalogName("catalog_test").
 				Precision(100).
+        Remarks("test column").
```

Comment:
The indentation looks off

Reply:
Fixed, thanks!
Looks like my editor mixed up spaces with tabs